ext/build_defaults.yaml

```diff
@@ -26,9 +26,9 @@ build_msi:
     repo: 'git://github.com/puppetlabs/hiera.git'
   sys:
     ref:
-      x86: '8db9d84da9950760144b5dfcd807213eecee4842'
-      x64: '12030f11e9bb2f085c68108bff34be6956b25df9'
-    repo: 'git://github.com/puppetlabs/puppet-win32-ruby.git'
+      x86: 'debafa740bca3f799151a3f59f7c028ac07a9bd3'
+      x64: '17e455a5be2743774c4357ee24770fe411ffb837'
+    repo: 'git://github.com/Iristyle/puppet-win32-ruby.git'
```

Comment:
These SHAs will need to be updated once the following PRs are merged:

https://github.com/puppetlabs/puppet-win32-ruby/pull/99 for 1.9.3-x86
https://github.com/puppetlabs/puppet-win32-ruby/pull/98 for 2.0.0-x64